Server/JokesOnYou.Web.Api/Services/UserService.cs

```diff
@@ -37,5 +44,40 @@ public async Task UpdateUser(UserUpdateDTO updateDTO)
         {
             throw new NotImplementedException();
         }
+
+        public async Task<UserReplyDTO> LoginUser(UserLoginDTO userLogin)
+        {
+            var user = await _userRepository.GetUserByEmail(userLogin.Email);
+            if (user != null)
+            {
+                var signInResult = await _signInManager.CheckPasswordSignInAsync(user, userLogin.Password, false);
+
+                if (signInResult.Succeeded)
+                {
+                    var userReplyDTO = new UserReplyDTO()
+                    {
+                        Id = user.Id,
+                        Email = user.Email,
+                        UserName = user.Email,
+                        Role = "",
```

Comment:
I still don't get why return `Role = ""`

Reply:
Can i just delete the role line? Not sure which roles we have defined so far

Reply:
Yeah you can just remove it, we arent handling roles right now